lib/stm32/f0/adc.c

```diff
@@ -519,9 +519,9 @@ void adc_disable_analog_watchdog(uint32_t adc)
  * @param[in] threshold Unsigned int8. Upper threshold value
```

Comment:
You need to update doxygen threshold type parameter to uint16_t as well (or preferably get rid of parameter type in documentation) - in any case threshold is not int8 anymore.

Reply:
done